### PR TITLE
ISS CDN configuration

### PR DESCRIPTION
**Pull request by Lennonka**
Cherry-pick into:

* [x] Foreman 3.2
* [x] Foreman 3.1

### CODE REVIEW

File: guides/common/modules/proc_configuring-satellite-for-air-gapped-operation.adoc [hunk 1]
```diff
@@ -0,0 +1,11 @@
+[id="configuring-satellite-for-air-gapped-operation_{context}"]
+= Configuring Satellite for Air-Gapped Operation
```
Comment (Lennonka) [inline guides/common/modules/proc_configuring-satellite-for-air-gapped-operation.adoc:2]: I'm pretty sure that this title doesn't say what we need to say here. Any ideas?

File: guides/common/modules/proc_configuring-satellite-for-air-gapped-operation.adoc [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+If you deployed your {ProjectServer} as  air-gapped, disable automatic communication with the Red Hat CDN to avoid checking what content is available to save system resources.
```
Comment (melcorr) [inline guides/common/modules/proc_configuring-satellite-for-air-gapped-operation.adoc:4]: Double space here.

**Comment by jturel** [comment]
I need to chat w/ @parthaa but I'm seeing https://github.com/theforeman/foreman-documentation/blob/master/guides/common/modules/proc_configuring-satellite-to-synchronize-content-with-a-local-cdn-server.adoc for the first time. This workflow ("local cdn server") is no longer applicable in combination with the two new ISS workflows. It should (imo) use ISS Export Sync which implies that they'd need to use a second system to export the content. Which means that doc ultimately goes away. Considering content ISOs are going away at some point in the future this is especially true. Partha can correct me if I'm forgetting something here

**Comment by jturel** [comment]
Update on my previous comment: The Red Hat CDN url can still be changed via the UI but those docs should likely be re-worked soon to reflect the deprecation of content ISOs. A docs bug seems appropriate as I'm not sure of the timeline of that deprecation

File: guides/common/modules/proc_configuring-satellite-for-air-gapped-operation.adoc (hunk 1 above)
Comment (melcorr) [reply guides/common/modules/proc_configuring-satellite-for-air-gapped-operation.adoc:2]: We will not really call it Satellite but {Project} right?

**Comment by Lennonka** [comment]
@melcorr I appreciate your early feedback. This PR is in the DRAFT mode. I was hoping that it was enough to say that as such, it is not yet ready for community review. Should I communicate this state of art differently to make it clearer next time?

**Comment by melcorr** [comment]
> @melcorr I appreciate your early feedback. This PR is in the DRAFT mode. I was hoping that it was enough to say that as such, it is not yet ready for community review. Should I communicate this state of art differently to make it clearer next time?

I would prefer if we didn't use vendor terms in the upstream at any point, if they can be avoided @Lennonka

**Comment by Lennonka** [comment]
@melcorr Oh, sure. Noted.

**Comment by Lennonka** [comment]
> I need to chat w/ @parthaa but I'm seeing https://github.com/theforeman/foreman-documentation/blob/master/guides/common/modules/proc_configuring-satellite-to-synchronize-content-with-a-local-cdn-server.adoc for the first time. This workflow ("local cdn server") is no longer applicable in combination with the two new ISS workflows. It should (imo) use ISS Export Sync which implies that they'd need to use a second system to export the content. Which means that doc ultimately goes away. Considering content ISOs are going away at some point in the future this is especially true. Partha can correct me if I'm forgetting something here

@jturel Let's consider the bigger picture:

1. I'm wondering if the sections about disconnected/air-gapped scenarios and about configuring SatServer for ISS Network Sync should actually be in this guide instead of this section and not in the Content Management Guide (CMG), because it looks more like a one-time configuration task than a repeated content-management task.
2. If so, the ISS chapter in the CMG could stil be only about ISS Export Sync with some tweaks (also of the heading) and definitely hyperlinked from the air-gapped config section in this "Installing Disconnected" Guide.

This is, of course, a part of the bigger problem that we've discussed, regarding guidance on multi-server setups in general.
I'll discuss it with our Content Strategist, but I'd like to know what you think as well.

**Comment by jturel** [comment]
> 1. I'm wondering if the sections about disconnected/air-gapped scenarios and about configuring SatServer for ISS Network Sync should actually be in this guide instead of this section and not in the Content Management Guide (CMG), because it looks more like a one-time configuration task than a repeated content-management task.
> 2. If so, the ISS chapter in the CMG could stil be only about ISS Export Sync with some tweaks (also of the heading) and definitely hyperlinked from the air-gapped config section in this "Installing Disconnected" Guide.
> 
> This is, of course, a part of the bigger problem that we've discussed, regarding guidance on multi-server setups in general. I'll discuss it with our Content Strategist, but I'd like to know what you think as well.

I think this was the missing piece as to what was bothering me about the existing docs - it mixes both satellite configuration/setups and also the content management aspects of those setups. Splitting the two seems absolutely correct.

**Pull request closed without merging**